lib/base-compiler.js

```diff
@@ -716,6 +720,29 @@ export class BaseCompiler {
         return this.getOutputFilename(dirPath, outputFilebase, key);
     }
 
+    async generateGnatDebug(inputFilename, options) {
+        const gnatDebugPath = this.getGnatDebugOutputFilename(inputFilename);
+
+        const newoptions = options.concat (['-gnatD']);
+
+        const execOptions = this.getDefaultExecOptions();
+        // A higher max output is needed for when the user includes headers
```

Comment:
(flyby) - is this a copy/pasted comment? Or does GNAT have lots of headers?

Reply:
Shameful copy/paste. Not really sure how to set this. I guess reusing this can be a good start.